src/modules/sv_detection_methods/analyze_split_read_method.cpp

```diff
@@ -28,7 +28,9 @@ void retrieve_aligned_segments(std::string const & sa_string, std::vector<Aligne
         split_string(sa_tag, fields, ',');
         if (fields.size() == 6)
         {
-            std::string ref_name = fields[0];
+            // As the chromosome names can differ between "1" and "chr1", we would distinguish same SVs from different
+            // input files if the chromosome naming is different. Thus we add the "chr" prefix.
+            std::string const & ref_name = (fields[0].starts_with("chr") ? fields[0] : ("chr" + fields[0]));
```

Comment:
This is not general enough I think. Imagine a reference name like "NC_000913.3 Escherichia coli str. K-12 substr. MG1655, complete genome" ([found here](https://www.ncbi.nlm.nih.gov/nuccore/NC_000913.3?report=fasta)), which doesn't start with `chr` and would turn into `chrNC_000913.3`. 
What we want to search is a single X or Y or at leat one digit, which is the regular expression `^([XY]|\d+)$`.

Reply:
A temporary value like `"chr" + fields[0]` must be stored in a variable and not a reference.

Reply:
I have a BAM file where you read exactly this in the header:
```
@SQ	SN:chr1	LN:248956422
@SQ	SN:chr2	LN:242193529
...
@SQ	SN:chr22	LN:50818468
@SQ	SN:chrX	LN:156040895
@SQ	SN:chrY	LN:57227415
@SQ	SN:chrM	LN:16569
@SQ	SN:chr1_KI270706v1_random	LN:175055
@SQ	SN:chr1_KI270707v1_random	LN:32032
...
@SQ	SN:chr22_KI270739v1_random	LN:73985
@SQ	SN:chrY_KI270740v1_random	LN:37240
@SQ	SN:chrUn_KI270302v1	LN:2274
@SQ	SN:chrUn_KI270304v1	LN:2165
@SQ	SN:chrUn_KI270303v1	LN:1942
...
@SQ	SN:chrUn_GL000218v1	LN:161147
@SQ	SN:chrEBV	LN:171823
```
So it is "chr" infront of everything.
(long_reads: HG002_PacBio_GRCh38.bam)
And I had a look into my other long read BAM files, and there was no mixed version (chr for 1 to Y and no chr for the others). Checking the short read files at the moment, but this will take a while.

Reply:
In the short read BAM I found
```
@SQ	SN:chrUn_JTFH01001996v1_decoy	LN:2009	AS:GRCh38_full_plus_hs38d1_analysis_set_minus_alts
@SQ	SN:chrUn_JTFH01001997v1_decoy	LN:2003	AS:GRCh38_full_plus_hs38d1_analysis_set_minus_alts
@SQ	SN:chrUn_JTFH01001998v1_decoy	LN:2001	AS:GRCh38_full_plus_hs38d1_analysis_set_minus_alts
```